### ISSUE TITLE
Editor: this image has an empty alt attribute

### ISSUE THREAD

**Issue by catkiinson** [Automattic/wp-calypso#31249]
#### What happened
Block editor suddenly started reporting a missing Alt text attribute on images, so it stopped displaying them in the editor and on the site. The image URLs themselves still worked fine.

#### Steps to fix 
- Used the Media Library Assistant plugin to bulk edit images adding Alt text. This resolved the issue for all the images apart from one page which had images in reusable column blocks. 
- The images were using the staging URL - ran command in CLI: search-replace. 
- All images now displaying correctly. 

#### Screenshot / Source
hc#10963669
#user-report
<img width="973" alt="screen shot 2019-03-06 at 6 46 42 am" src="https://user-images.githubusercontent.com/43645273/53889674-a4fde080-3fdb-11e9-99bf-d5d4a5d86ec0.png">

**Comment by gwwar** [comment]
cc @Automattic/gutenpack not sure if this is a block related issue or an issue with the integration. Have y'all see anything similar?

**Comment by simison** [comment]
> Have y'all see anything similar?

Nope, this is new to us.

@catkiinson so alt tags were gone when the post was opened in the editor?

**Comment by simison** [comment]
What type of site was this? Mention of plugins makes me think atomic?

**Comment by sirreal** [comment]
Seems to be from Gutenberg:

```js
} else if ( filename ) {
	defaultedAlt = sprintf( __( 'This image has an empty alt attribute; its file name is %s' ), filename );
} else {
	defaultedAlt = __( 'This image has an empty alt attribute' );
}
```

https://github.com/WordPress/gutenberg/blob/c29b729a6d107fc5a7ef53efafda126d6fbb68e0/packages/block-library/src/image/edit.js#L578-L582

**Comment by sirreal** [comment]
> What type of site was this? Mention of plugins makes me think atomic?

Atomic

**Comment by joweber123** [comment]
#13098327-hc

Another report of this on an Atomic site.  User had an image with no Alt text displaying this same error.  I cannot seem to find a reason for this though as some other images on their site also have no Alt-text but are displaying normally.

**Comment by joweber123** [comment]
#2070634-zen

Another report of this on a simple site.

**Comment by katinthehatsite** [comment]
I had another site that seems to have a similar issue in #hc-13066519

View from the editor

<img width="834" alt="Screen Shot 2019-07-15 at 8 46 59 AM" src="https://user-images.githubusercontent.com/25575134/61219363-0e0c0800-a6e2-11e9-81c4-87fa4bb2e1d9.png">

View on the site

<img width="501" alt="Screen Shot 2019-07-15 at 8 47 10 AM" src="https://user-images.githubusercontent.com/25575134/61219386-1c5a2400-a6e2-11e9-8f06-684670822010.png">

**Comment by fernmed** [comment]
Just had a user with this same issue. Images all of a sudden started disappearing from the page, and in their editor they say saw the error. 

![https://cld.wthms.co/iA6mLP](https://cld.wthms.co/iA6mLP+) 
Image link: https://cld.wthms.co/iA6mLP

AT Site

#14496797-hc

**Comment by vedmin** [comment]
Faced the same issue and can not find any solution on the web.

**Comment by thebud15** [comment]
Reported in #2284239-zen

**Comment by noteeth** [comment]
Also reported in #2289708-zen

**Comment by fresatomica** [comment]
The same issue on a simple site: https://en.forums.wordpress.com/topic/cant-see-the-image-on-my-wordpress-site/

**Comment by rossanafmenezes** [comment]
Another report HC-17812927

The session expired so I had to open the user's account again and now there's no message, only the image missing:

![](https://cldup.com/Woe9hCzWlM.png) 

But it was displayed the same message 5 minutes ago saying the image has a missing attribute etc

**Comment by nagpai** [comment]
#24578859-hc

**Comment by dolgelukkig** [comment]
Same issue, simple site, Premium plan
Site: http://jackielunntextiles.com/
[#6596915-hc](https://mc.a8c.com/support-stats/happychat/transcript.php?id=6596915)
screenshot:
![Markup on 2020-11-09 at 12:40:52](https://user-images.githubusercontent.com/5507879/98537036-f66a7280-2288-11eb-81c1-572131cf10ad.png)

**Comment by ivan-ottinger** [comment]
Another report: 3518993-zd-woothemes / Simple site (Free)

The user mentioned that the issue occurs on the images that were pasted into the editor (in their case from Facebook).
→ I suggested them to upload images directly until the issue is fixed (in case it is somehow related to the pasting).

**Comment by nerdysandy** [comment]
I can't replicate it. Also, when copy/pasting images from Facebook. For an image that doesn't have alt text, it adds it automatically https://d.pr/i/FrGQQH I tested this using Image and Media & Text block.

**Comment by lawlsh** [comment]
Under the Block tab/Image Size, Select Medium and the image will show. Selecting Thumbnail and Full Size somehow caused the image not to display.

**Comment by manuelrebollo-a11y** [comment]
Reported, and replicated in 5328327-zd-woothemes
Workaround detailed in p1656288117149629/1656285431.422419-slack-C03TY6J1A

**Comment by druesome** [comment]
In addition to @manRebollo 's comment, I'm adding my findings here that can hopefully help:

It appears that the issue affects only some thumbnails generated from the original image. 

Thumbnails that don't work (returns a 504 Gateway Time-out error):

- https://reblogtest.blog/wp-content/uploads/2022/06/screencast-1200x750.gif
- https://reblogtest.blog/wp-content/uploads/2022/06/screencast-1024x640.gif

Thumbnails that work:

- https://reblogtest.blog/wp-content/uploads/2022/06/screencast-768x480.gif
- https://reblogtest.blog/wp-content/uploads/2022/06/screencast-600x375.gif
- https://reblogtest.blog/wp-content/uploads/2022/06/screencast-300x188.gif

Original image: https://reblogtest.blog/wp-content/uploads/2022/06/screencast.gif (loads without issue)

Regenerating thumbnails in CLI via `media regenerate 3039`  or manually adding ALT text in the image settings didn't help get the image to load.  However, selecting Image size > Full Size/Medium/Thumbnail in the block settings gets the image to show up.  It's the Large setting that does not work.

<img src="https://d.pr/i/5nOy91+" width="300" />

**Comment by marianearchimbaud** [comment]
I am facing the same situation. This is a Simple Site with a Premium plan. 35911276-hc
Screenshot:
<img width="1065" alt="Screenshot on 2022-06-27 at 08-59-27" src="https://user-images.githubusercontent.com/72237737/175959563-1e7cdaae-3a1f-4f16-ae81-4a5f2a120b9d.png">

This is not happening with all images though.
These are the spec of the image that shows the error:
FILE NAME | sift_final-2.png
-- | --
PNG
3.73 MB
2000 ✕ 31667
27 June 2022

I am not quite sure if the error depends on the size of the image.

I tried to convert the image to JPG to see if that help, but I had no luck with that.

The user is having issue with this image file too:
FILE NAME | nwn_present.png
-- | --
PNG
23.74 MB
2500 ✕ 28125
26 June 2022

**Comment by jromales** [comment]
Another report in 5421007-zd-woothemes

![Markup 2022-07-25 at 09 41 04](https://user-images.githubusercontent.com/89536719/180968257-939f8745-d94d-407d-ad09-ca0c57ce0305.png)

The workaround was to replace all corrupted images with the same image from Media Library

**Comment by eduardozulian** [comment]
Also reported in 6688692-zen

**Comment by ash1eygrace** [comment]
Encountered here: 

- p9F6qB-dQs-p2
- 7393747-zd-a8c

**Comment by MariannevdS** [comment]
> In addition to @manRebollo 's comment, I'm adding my findings here that can hopefully help:
> 
> It appears that the issue affects only some thumbnails generated from the original image.
> 
> Thumbnails that don't work (returns a 504 Gateway Time-out error):
> 
>     * https://reblogtest.blog/wp-content/uploads/2022/06/screencast-1200x750.gif
> 
>     * https://reblogtest.blog/wp-content/uploads/2022/06/screencast-1024x640.gif
> 
> 
> Thumbnails that work:
> 
>     * https://reblogtest.blog/wp-content/uploads/2022/06/screencast-768x480.gif
> 
>     * https://reblogtest.blog/wp-content/uploads/2022/06/screencast-600x375.gif
> 
>     * https://reblogtest.blog/wp-content/uploads/2022/06/screencast-300x188.gif
> 
> 
> Original image: https://reblogtest.blog/wp-content/uploads/2022/06/screencast.gif (loads without issue)
> 
> Regenerating thumbnails in CLI via `media regenerate 3039` or manually adding ALT text in the image settings didn't help get the image to load. However, selecting Image size > Full Size/Medium/Thumbnail in the block settings gets the image to show up. It's the Large setting that does not work.
> 
> <img alt="" width="300" src="https://camo.githubusercontent.com/b5f2617c2381f39600035af242ce3bae01dddcf618f8c1c24300af2caff9d5db/68747470733a2f2f642e70722f692f356e4f7939312b">

This 'fix' also helps me with the same issue.
Using wordpress twenty twentyfour theme
Pattern: offset gallery 4 columns

All seemed to work fine 
I moved the website to new location, all still worked fine and then  at some point in the week after in the editor the thumbnails dissapeared and in stead i got the text about the empty alt-attribute.
Front-end no problem but in editor de thumbs were missing. Jiggling the resolution from 'large' to 'full' brought the image back. Changing back to 'large" image seems to stay. (Don't know for how long though)

**Comment by Mfarroq1** [comment]
7952355-zen

**Comment by mariammcWP** [comment]
Newer and related: https://github.com/Automattic/wp-calypso/issues/87642

**Comment by radtechgh** [comment]
8495091-zen
